docs/tutorial/navigation-history.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Navigation History"
+description: "The NavigationHistory API allows you to manage and interact with the browsing history of your Electron application."
+slug: navigation-history
+hide_title: false
+---
+
+# Navigation History
+
+## Overview
+
+The [NavigationHistory](../api/navigation-history.md) API allows you to manage and interact with the browsing history of your Electron application. This powerful feature enables you to create intuitive navigation experiences for your users.
+
+### Accessing NavigationHistory
+
+To use NavigationHistory, access it through the [webContents](../api/web-contents.md) of your [BrowserWindow](../api/browser-window.md):
```

Comment:
```suggestion
Navigation history is stored per [`WebContents`](../api/web-contents.md) instance. To access a specific instance of the NavigationHistory class, use the WebContents class' [`contents.navigationHistory` instance property](https://www.electronjs.org/docs/latest/api/web-contents#contentsnavigationhistory-readonly).
```

I think this is a bit clearer since this is not only limited to BrowserWindow APIs but anywhere a `WebContents` instance exists.